ramls/types/resources/resourceDataAttributes.json

```
@@ -6,6 +6,15 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "alternateTitles": {
+      "type": "array",
+      "description": "Alternate Title Schema",
```

```suggestion
      "description": "Alternate Titles Schema",
```

---

I have another idea:
```suggestion
      "description": "List of alternate titles",
```